tests/realworld.rs

```diff
@@ -51,6 +48,7 @@ mod realworld {
     }
 
     #[test]
+    #[cfg(linux)]
```

Comment:
🤔 could I just be using `#[cfg(linux]` instead of `#[cfg(all(unix, not(target_os = "macos")))]`?
or does that not work on bsd?

Reply:
cfg(linux) doesn't work the same way as the unix one unfortunately, I'll review this usage because this might be a wrong macro usage.